doc/api/fs.md

```diff
@@ -476,11 +476,14 @@ Reads data from the file and stores that in the given buffer.
 If the file is not modified concurrently, the end-of-file is reached when the
 number of bytes read is zero.
 
-#### `filehandle.readableWebStream()`
+#### `filehandle.readableWebStream([options])`
 
 <!-- YAML
 added: v17.0.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/00000
```

Comment:
I wish the GitHub API somehow simplified getting line numbers from patches, allowing to make ``s/PRNUMBER/`${PRNUMBER}`/`` suggestions fully automated...
```suggestion
    pr-url: https://github.com/nodejs/node/pull/58548
```